### PR TITLE
add Simultaneous Attack plugin

### PR DESCRIPTION
**Pull request by Bulmenisaurus**
added my "Simultaneous Attack" plugin :)

### CODE REVIEW

File: content/productivity/simultaneous-attack/plugin.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    planetElement.innerText = `L${planet.planetLevel}R${planet.upgradeState.reduce((a, b) => a + b, 0)} ${pg.getPlanetName(planet)}`;
+    planetElement.title = locationId;
+    planetElement.style.textDecoration = "underline";
+    planetElement.style.background = "none";
+    planetElement.style.border = "none";
+    planetElement.style.color = "white";
+    planetElement.style.outline = "none";
+    planetElement.style.padding = "0";
+    if (clickable) {
+        planetElement.addEventListener("click", () => {
```
Comment (jacobrosenthal) [inline content/productivity/simultaneous-attack/plugin.js:28]: You addeventlistener quite often, but I dont see them being removed at all.

Comment (Bulmenisaurus) [reply content/productivity/simultaneous-attack/plugin.js:28]: Ok! I'll make the event listeners all methods of the class so I can cancel them. They should be removed on the destroy method, right?

File: content/productivity/simultaneous-attack/plugin.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+            return { energyArriving: 0, damageDealt: 0 };
+        let energyArriving = 0;
+        for (const planet of this.planetAttackers) {
+            energyArriving += df.getEnergyArrivingForMove(planet, this.planetTarget, void 0, df.getPlanetWithId(planet).energy * (this.planetEnergySendingPercent / 100));
+        }
+        const damageDealt = energyArriving / (df.getPlanetWithId(this.planetTarget).defense / 100);
+        return { energyArriving, damageDealt };
+    }
+    // displays the damage calculations from above on a container
+    updateDamageCalculations(container) {
```
[the remark this answers is not in the document]
Comment (Bulmenisaurus) [reply content/productivity/simultaneous-attack/plugin.js:183]: Should I set a `setInterval` to do the calculations then?

File: content/productivity/simultaneous-attack/plugin.js (hunk 1 above)
Comment (jacobrosenthal) [reply content/productivity/simultaneous-attack/plugin.js:28]: It seems like for these, they should be removed when you click the 'clear' button?

File: content/productivity/simultaneous-attack/plugin.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jacobrosenthal) [reply content/productivity/simultaneous-attack/plugin.js:183]: Yeah thats an option

File: content/productivity/simultaneous-attack/plugin.js (hunk 1 above)
Comment (Bulmenisaurus) [reply content/productivity/simultaneous-attack/plugin.js:28]: The `clear` button is for clearing the attackers, not event listeners :) I should probably make that more clear!

Comment (jacobrosenthal) [reply content/productivity/simultaneous-attack/plugin.js:28]: Oh I understand, but youre creating a listener for each attacker. Im not honestly sure its a problem, but you're clearing the list, you could clear the handlers too

**Pull request merged**